src/renderers/shared/fiber/ReactFiberDevToolsHook.js

```diff
@@ -12,15 +12,30 @@
 
 'use strict';
 
+var emptyFunction = require('fbjs/lib/emptyFunction');
 var warning = require('fbjs/lib/warning');
 
 import type {Fiber} from 'ReactFiber';
 import type {FiberRoot} from 'ReactFiberRoot';
 
 declare var __REACT_DEVTOOLS_GLOBAL_HOOK__: Object | void;
 
+let didCatchErrors = false;
+function tryCall(fn, rendererID, arg) {
+  try {
+    fn(rendererID, arg);
+  } catch (err) {
+    // Catch all errors from DevTools because throwing might be unsafe
+    if (__DEV__) {
+      warning(!didCatchErrors, 'React DevTools encountered an error: %s', err);
```

Comment:
Right. If the first parameter is `false` the warning will be printed. You init `didCatchErrors` to `false` and then set it to `true` after calling `warning` but... you invert the value when you pass to warning (`!didCatchErrors`) so... won't this skip the first warning, and print every one after it? (Isn't that the opposite of what you want to do?)

Am I misreading this? 😅